pkg/helpers/helpers_test.go

```diff
@@ -221,3 +222,75 @@ func Test_DecodeSnowflakeAccountIdentifier(t *testing.T) {
 		require.ErrorContains(t, err, fmt.Sprintf("unable to read identifier: %s", id))
 	})
 }
+
+func TestParseRootLocation(t *testing.T) {
+	tests := []struct {
+		name     string
+		location string
+		id       string
+		path     string
+		wantErr  bool
```

Comment:
do we want to maybe expect a specific errors for different caseS?

Reply:
Done.